### DISCUSSION TITLE
Support for Fontawesome

### DISCUSSION THREAD

**Discussion by VanNostrand** [gohugo-ananke/ananke#727] [3 - Ideas]
Fontawesome is a widely used icon font (https://fontawesome.com/) and it is also used on websites. It would be nice to somehow easily use the font's icons in the markdown code.

Here someone wrote a blog post about using hugo with fontawesome, but it requires the user to write a couple of partials, which is why I have not tried it with Ananke... 
https://somethingstrange.com/posts/hugo-with-fontawesome/

It would be a great feature to natively support this somehow. I know new features don't have high priority at the moment, but one day maybe?

**Comment by davidsneighbour** [comment]
I have written a module to add any icon set easily via shortcode. That (adding a shortcode) would be an easy way to integrate it into Ananke without adding ballast to sites that don't use it. Let me look into it.
